harvester_e2e_tests/conftest.py

```diff
@@ -49,9 +50,42 @@ def check_depends(self, depends, item):
 DepMgr.checkDepend, DepMgr._check_depend = check_depends, DepMgr.checkDepend
 
 
+def merge_config(user_config_path, default_config_path):
+    """
+    Merge two config files. The user config takes precedence over the default
+    config.
+    """
+
+    try:
+        with open(user_config_path, 'r') as ucnf:
+            user_data = yaml.safe_load(ucnf)
+    except FileNotFoundError:
+        user_data = {}
+
+    try:
+        with open(default_config_path, 'r') as dcnf:
+            default_data = yaml.safe_load(dcnf)
+    except FileNotFoundError:
+        default_data = {}
+
+    config = {}
+    for key in default_data:
+        config[key] = user_data.get(key, default_data[key])
+
+    for key in user_data:
+        if not key in default_data.keys():
+            config[key] = user_data[key]
+
+    return config
+
+
 def pytest_addoption(parser):
-    with open('config.yml') as f:
-        config_data = yaml.safe_load(f)
+    user_home = os.getenv('HOME')
+    user_config_path= os.path.join(user_home, '.config', 'harvester', 'tests',
```

Comment:
IMO it's better to use an env. vairable like `HVST_TESTS_CONFIG_PATH` to give user the flexibility instead of fixed path.
e.g. `user_config_path = os.getenv('HVST_TESTS_CONFIG_PATH')`

Reply:
Good point. I've added the variable `HARVESTER_TESTS_CONFIG` as possible location for the config file.
I still want it to look in a default location outside of the repository, since this way dropping a file somewhere is enough and the user doesn't need to set an environment variable as well.
I've also put a short line in the readme to explain all this.